pages/osx/whence.md

```diff
@@ -0,0 +1,19 @@
+# whence
+
+> A zsh builtin to indicate how a given command would be interpreted.
+
+- Interpret {{command}}, with expansion if defined as an `alias` (similar to `command -v` builtin):
+
+`whence {{command}}`
+
+- Display type of {{command}}, with location if defined as a function, or binary (equivalent to `type` and `command -V` builtins):
```

Comment:
Perhaps this will read better?
```suggestion
- Display type of {{command}}, with location if defined as a function, or binary (equivalent to the `type` and `command -V` builtins):
```